### ISSUE TITLE
Resigning options, Splitting pkgs and ISOs, SFO info displayed differently.  

### ISSUE THREAD

**Issue by DeViL303** [pink1stools/PS3gbs#7]
So I was thinking about what you said about having global options for HAN or CFW resigning, And I'm not sure if its the best way, The user might just want to resign one package for HAN for example, or they might have HAN and CFW systems, then it might not be great to have to go into the settings to check/change the setting every time.

How about having a button in the main screen like this, instead of having 2 buttons, the button could change to only offer one option depending on what pkg type you have selected, if its a debug (CFW) pkg offer HAN resigning, if its a HAN resigned pkg offer CFW resigning:

[img]https://i.imgur.com/rAbEs15.png [/img]

Also how about including options to split pkgs into 4GB chunks, Could also have an option to split ISOs into 666 chunks too?

Also it would cool if the SFO box could show more human readable info , so for example instead of showing "Bootable: 1" , show "Bootable: Yes", instead of "Category: AV", show "Category: Video (AV), instead of "Resolution: 29" , show "Resolution: 1080p, 720p" etc..

**Comment by xXxTheDarkprogramerxXx** [comment]
I Agree with @DeViL303 
@pink1stools  having buttons for each option would be better than a global setting as i have a HAN system and a DEX system myself 

Moving this to enhancements and assigning to either of us to complete

**Comment by pink1stools** [comment]
I agree too. The more we work I realize some of the ideas we first had just aren't the best way.
On the SFO viewer do we want it to be editable or should we do a separate editor that will auto rebuild the game when saving?
